assets/js/frontend-form.js

```diff
@@ -833,43 +833,43 @@
 
         editorLimit: {
 
-            bind: function(limit, field, type) {
+            bind: function(limit, limit_type, field, type) {
```

Comment:
Do no add new param in between old params. Always add new param as a last argument. Change the position of `limit_type`, move it to after `type` param and change your code accordingly.